tools/acrn-crashlog/Makefile

```diff
@@ -0,0 +1,80 @@
+#
+# ACRN-Crashlog Makefile
+#
+
+BASEDIR 	:= $(shell pwd)
+OUT_DIR 	?= $(BASEDIR)
+BUILDDIR	:= $(OUT_DIR)/acrn-crashlog
+CC		:= gcc
+RM		= rm
+
+CFLAGS	:= -Wall -Wextra -pedantic
+CFLAGS	+= -m64 -D_GNU_SOURCE -DDEBUG_ACRN_CRASHLOG
```

Comment:
What is this flag `DEBUG_ACRN_CRASHLOG` doing? And should there be a `RELEASE` equivalent to it?

Reply:
DDEBUG_ACRN_CRASHLOG is a flag for debug purpose, where we can  output more log information, like func and line.

Reply:
The tool can be compiled without the DEBUG flag. Could I know what the meaning of you mentioned RELEASE mode? Is that mode to remove all the debug flag? If so, there will be no problem when remove this flag.

Reply:
My point is that you unconditionally compile the tool using the DEBUG flag so unless somebody comes in a manually edits this Makefile, all debug info is included. As an example, the hypervisor's Makefile uses a variable called `RELEASE` to determine some compilation options. I think it would be more appropriate to have this in place rather than unconditionally include the debug option. We could either use the same `RELEASE` variable. Thoughts?

Reply:
Thanks a lot for your comments, @gvancuts . The Makefiles has been updated according to your comments, please help to take a review. Thanks in advanced.

+CFLAGS	+= -m64 -D_GNU_SOURCE
+ifeq ($(RELEASE),0)
+CFLAGS	+= -DDEBUG_ACRN_CRASHLOG
+endif

Reply:
Thanks @chengangc !